### PR TITLE
chore: upgrade vitepress dependency to stable release and adjust configuration

### PR DESCRIPTION
**Pull request by colinscz**
**What kind of change does this PR introduce?** (check at least one)
Upgrading Vitepress to the first major release and fix import.meta.glob issue: https://github.com/vitejs/vite/discussions/7066

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [x] Other, please describe:
   - [x] Styling changes, theme updates to CSS-Variables used 
   - [x] https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css

**The PR fulfills these requirements:**

- [x] It's submitted to the `main` branch for v2.x (or to a previous version branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included

**Other information:**

### CODE REVIEW

**Review by aggieN — DISMISSED** [review]

File: docs/.vitepress/theme/index.ts [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 import { h } from 'vue';
-import Theme from 'vitepress/theme';
+import DefaultTheme from 'vitepress/theme';
 import './style.css';
 import { defineCustomElements } from '@six-group/ui-library/loader';
+import { Theme } from 'vitepress';
 
 // @ts-ignore
-const modules = import.meta.globEager('../../examples/**/*.vue');
+const modules = import.meta.glob("../../examples/**/*.vue'", { eager: true });
```
Comment (ibirrer) [inline docs/.vitepress/theme/index.ts:9]:
```suggestion
const modules = import.meta.glob('../../examples/**/*.vue', { eager: true });
```

**Review by ibirrer — CHANGES_REQUESTED** [review]

File: docs/.vitepress/theme/style.css
Comment (ibirrer) [inline docs/.vitepress/theme/style.css:1]: Borders should probably have the same color as the content:
image

File: docs/.vitepress/theme/style.css [hunk 2]
```diff
@@ -10,30 +10,28 @@
...
+  --vp-code-color: #476582;
 }
 
 /**
  * Component: Button
  * -------------------------------------------------------------------------- */
 
 :root {
-  --vp-button-brand-border: var(--vp-c-brand-light);
+  /*  --vp-button-brand-border: var(--vp-c-brand-2);*/
```
Comment (ibirrer) [inline docs/.vitepress/theme/style.css:28]: better remove than comment

File: docs/.vitepress/theme/index.ts [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 import { defineCustomElements } from '@six-group/ui-library/loader';
+import { Theme } from 'vitepress';
 
 // @ts-ignore
-const modules = import.meta.globEager('../../examples/**/*.vue');
+const modules = import.meta.glob('../../examples/**/*.vue', { eager: true });
 const exampleComponents = [];
 for (const path in modules) {
   exampleComponents.push(modules[path].default);
+  console.log(path);
```
Comment (ibirrer) [inline docs/.vitepress/theme/index.ts:13]: remiove console.log

**Review by ibirrer — CHANGES_REQUESTED** [review]

**Review by ibirrer — APPROVED** [review]

File: docs/.vitepress/theme/style.css
Comment (ibirrer) [inline docs/.vitepress/theme/style.css:1]: Search seems broken:
image

**Review by ibirrer — APPROVED** [review]

File: docs/.vitepress/theme/style.css (hunk 2 above)
Comment (colinscz) [reply docs/.vitepress/theme/style.css]: 936ea8ccb912e8cc1c0db2ad666f6cab49b1342c

File: docs/.vitepress/theme/index.ts (hunk 3 above)
Comment (colinscz) [reply docs/.vitepress/theme/index.ts]: 936ea8ccb912e8cc1c0db2ad666f6cab49b1342c

File: docs/.vitepress/theme/style.css
Comment (colinscz) [reply docs/.vitepress/theme/style.css:1]: 749bcb979c6092d94b1de5771071247ff98cf60e

Comment (colinscz) [reply docs/.vitepress/theme/style.css:1]: 749bcb979c6092d94b1de5771071247ff98cf60e

**Review by pennal — APPROVED** [review]

**Pull request merged**